rules/no-disabled-tests.js

```diff
@@ -1,19 +1,16 @@
 'use strict';
 
 function getName(node) {
-  function joinNames(a, b) {
-    return a && b ? a + '.' + b : null;
-  }
-
   switch (node && node.type) {
     case 'Identifier':
       return node.name;
     case 'Literal':
       return node.value;
     case 'MemberExpression':
-      return joinNames(getName(node.object), getName(node.property));
+      return `${getName(node.object)}.${getName(node.property)}`;
   }
 
+  /* istanbul ignore next */
   return null;
```

Comment:
Is this just to be future proof?

Reply:
This is for any unmatched node-types (ie those we can't find names for), eg
```
(a || b).f(); // callee.object here is LogicalExpression
```

Reply:
Great, thanks! Added those 2 examples, and coverage hit 100% 🙂

Reply:
:+1: